packages/checkout-com/nuxt/index.js

```diff
@@ -3,9 +3,7 @@ import path from 'path';
 export default function CheckoutComModule(moduleOptions) {
```

Comment:
does it have anything to do with CKO?

Reply:
Damn. Looks like I switched branch to old one while I was working on the next task by mistake. I will fix it first thing tomorrow

Reply:
Fixed